src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
+import { BrowserRouter } from "react-router-dom";
```

Comment:
import { BrowserRouter as Router }... then call Router instead of BrowserRouter

Reply:
Ok I am going to put it into App.js too as suggested